core/src/packet.rs

```diff
@@ -75,6 +91,17 @@ impl PartialEq for Packet {
     }
 }
 
+impl Packable for Packet {
+    type Metadata = Meta;
+    fn get_meta(&self) -> &Meta {
```

Comment:
get()ters without parameters are named the same as the attribute in the Solana tree (generally ;) )

get_xxx() -> xxx()